openpathsampling/features/numpydocparse.py

```diff
@@ -0,0 +1,145 @@
+class NumpyDocParser(object):
```

Comment:
(On `NumpyDocTools`): The docstring you added is good. Also, I just wanted to mention that this is a pretty cool trick. If only there was a way to get sphinx to read such dynamically-created docstrings (I assume it doesn't, at least -- we can't use the `SnapshotFactory` and get documentable default snapshots, right?)